src/pages/oferta.tsx

```diff
@@ -10,43 +10,50 @@ const OffertPage = () => {
   return (
     <ParallaxProvider>
       <Layout
-        yellowCircle={true}
-        yellowCircle2={true}
-        orangeHex={true}
-        grayTriangle={true}
-        pinkTriangle={true}
-        blueHex={true}
-        redCircle={true}
-        yellowSquare={true}
-        blueSquare={true}
-        heroBG={true}
-        firstMidBG={true}
+        yellowCircle={false}
+        yellowCircle2={false}
+        orangeHex={false}
+        grayTriangle={false}
+        pinkTriangle={false}
+        blueHex={false}
+        redCircle={false}
+        yellowSquare={false}
+        blueSquare={false}
+        heroBG={false}
+        firstMidBG={false}
```

Comment:
Maybe we can do it in a smarter way and set all of those properties as default: false in the Layout component so we don't need to pass false all across the app, but only if it is true

Reply:
This is what I get from chatGPT question about handling default props:

In your current code, you're manually merging the default props with the incoming props inside the component:

```typescript
const layoutElementsProps: ILayoutElements = { ...defaultProps, ...layoutElements };
```

While this works, it's not considered a best practice in React. Instead, React has a built-in mechanism to handle default props. 

From React version 16.6.0 and onwards, you can use `defaultProps` on a function component:

```typescript
Layout.defaultProps = defaultProps;
```

However, starting from React version 17, the `defaultProps` on function components are being deprecated. So, the recommended way to set default props in a function component is to use default parameters in the function signature:

```typescript
const Layout: React.FC<Props> = ({
  children,
  screenComponent,
  isSubPage,
  isContactPage,
  footerWhiteBg,
  ...layoutElements
}: Props = defaultProps) => {
  // code
};
```

This way, if a prop isn't provided when the component is used, it will default to the value specified in `defaultProps`. 

One other thing worth mentioning is that if `ILayoutElements` is not used elsewhere, it might be better to define it in the same file as `Layout`, rather than importing it, for better readability and maintainability. 

Lastly, it's generally a good idea to keep components as small and focused as possible. If `Layout` is getting too big or handling too many props, consider splitting it into smaller, more focused components.